### DISCUSSION TITLE
Psychosis can turn NPCs to Team2 and they get shot at, failing the mission

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14771] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Psychosis can turn NPCs to team2 (I guess they're going "crazy"), which also means other NPCs will fire on sight and potentially kill them.

For missions such as VIP transport, if the VIP npc goes crazy the mission can just casually fail while you're not looking, for example if you happen to drive near a watcher...
![image](https://github.com/user-attachments/assets/d3642d98-5404-49c9-998b-f7ccab551f08)
![image](https://github.com/user-attachments/assets/24ab5dc0-08fc-47c7-a429-fade2f6f9def)

### Reproduction steps

1. start campaign
2. unlockmission escort2
3. giveaffliction psychosis 100 "vip name"
4. NPC will go crazy and security will shoot it

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
> for example if you happen to drive near a watcher...

Yes, don't

**Comment by SomeRandomNoobKekeke** [comment]
Also: Do you have a chance to deliver them handcuffed or outpost just won't accept deranged passengers?

**Reply by SomeRandomNoobKekeke** [reply]
Ok, i tested:
If you deliver them handcuffed then mission will succeed, which is ok
Also passengers have to be gazed for like a minute to become insane, so you have time to kill the watcher
And they won't just attack you, they'll also shoot each other, which is funny

**Comment by Regalis11** [comment]
This is actually intentional: if the characters go psychotic, it can make them go hostile (and the crew to fight them). I see how this can be confusing though, so I guess we might need to rethink how this works or communicate it to the players more clearly. But I'll move this to the "ideas & suggestions" area, because this isn't really a bug.

**Reply by ubertpendragon** [reply]
You can leave the VIP in a room across the sub with its own security and they will gun it down if VIP goes psychotic. The mission will fail "at random".
